### ISSUE TITLE
<Burial Claim Confirmation Email> from <Benefits Non-Disability Team>

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#78035]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Benefits - Non-Disability Experience 
- **OCTO product owner:** Emily Theis
- **Product name:** Burial Benefits (21P-530EZ)
- **Product manager:** Laura Steele
- **Slack channel:** #benefits-non-disability
- **Dedicated content writer on your team (if you have one):** none
- **Dedicated a11y specialist on your team (if you have one):** none
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [x] Something else: Adding a confirmation email to the Burial Benefits form

### What's the nature of your initiative and desired outcomes?

As part of #65251 and #68984, we are updating the existing online Burial Form to align with the most recent pdf version of the form. A follow-on component of this work involves adding a confirmation email, so individuals who submit claims receive official confirmation of their claim submission and understand next steps.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

We have drafted the initial [copy for the email](https://docs.google.com/document/d/1EYAcOxEPykMe2CMo7a7zU1Alw6HcHd_8CSnkBiqce2s/edit?usp=sharing) and sent it to our Burial VBA SMEs on 3/11 requesting feedback.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [x] Midpoint review (for form updates)
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** #65251

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Email copy](https://docs.google.com/document/d/1EYAcOxEPykMe2CMo7a7zU1Alw6HcHd_8CSnkBiqce2s/edit?usp=sharing)
- [Burial Initiative Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/burials-memorials/burial-allowance/530-initiative-brief-2024-updates.md)
- [Form Design (Figma)](https://www.figma.com/file/jpJQSCKiGegacdW4RH5uQB/Burial-530-2024-Updates-MVP?type=design&node-id=174-43726&mode=design)

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by aliyahblackmore** [comment]
Just tracking some updates here:

- A and Aliyah made some adjustments to the [confirmation email](https://docs.google.com/document/d/1EYAcOxEPykMe2CMo7a7zU1Alw6HcHd_8CSnkBiqce2s/edit#heading=h.610a38gjnfbn)
- Aliyah will share the content via Fiorella's [SharePoint document](https://dvagov.sharepoint.com/sites/vaabdvro/Shared%20Documents/Forms/AllItems.aspx?csf=1&web=1&e=yy22V8&cid=5b52d860%2D05be%2D4972%2D9e04%2De40d1d209383&FolderCTID=0x01200073ECFC0A16DB51439C06C6FDC0E92947&id=%2Fsites%2Fvaabdvro%2FShared%20Documents%2FNon%2Ddisability%20Benefits&viewid=3fa7a9bb%2D3d4e%2D44c2%2Db93f%2D629268a08e72) on Teams next week
- Laura confirmed on 3/29 that the confirmation email will go out when the form is at 100% (late April)

**Comment by aliyahblackmore** [comment]
**Tracking updates here:**

- I track the confirmation email updates in the[ SharePoint doc from Fiorella](https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/_layouts/15/Doc.aspx?sourcedoc=%7B54EB63D6-4B97-4104-AFE2-F960BEA55870%7D&file=Burial_21P-530EZ%20Confirmation%20Email-Draft-1.0.docx&action=default&mobileredirect=true). 
- I shared the confirmation email adjustments with SMEs via Teams. One SME replied and confirmation adjustments were accurate and that the email looks good. I'll follow up on the thread later in the week to confirm that other SMEs had a chance to review.

**Comment by aliyahblackmore** [comment]
**Updates here:** 

- SMEs provided their feedback via SharePoint.
- Since Laura confirmed that this email will start to go out once the form is at 100%, I'll be sure to wrap up feedback and any last adjustment next week or the week after. I'll let Fiorella know via this ticket and Slack.

**Comment by aliyahblackmore** [comment]
### **530EZ confirmation email:** 

- We made recent updates to the 530EZ confirmation email to include section "How to submit supporting documents." We are still waiting on SMEs to review this section of the email. Laura Steele confirmed via Slack (4/29) that these updates are okay to happen post 100%. 

- There were other updates to this email that the product team wants to go out once the form is at 100%. That's all of the content **EXCEPT** for the section and content in "How to submit supporting documents" [This is a link to the SharePoint document from Fiorella](https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/Shared%20Documents/Non-disability%20Benefits/Burial/Design%20Artifacts/Burial_21P-530EZ%20Confirmation%20Email-Draft-1.0.docx?d=w54eb63d64b974104afe2f960bea55870&csf=1&web=1&e=tt9GWS). 
  - The content can be added to VA Notify - @steele-lm Fiorella asked me to tag you to do this. 
  - SMEs haven't replied to a recent message 4/29 about making these updates, **so before this email goes live, just confirm that they are okay with that.**

**A few other important notes:**

- SMEs did their last review of this email in April and there were no outstanding content items. There was one comment re: link text that I replied to.  

- I shared with SMEs today (4/29) that the product team is planning to update the email with the other changes (not including the How to submit supporting documents section) and send out the updated email once the online form is at 100%. I tagged you @steele-lm and @aprocik1 and let them know that if they have concerns with you all publishing these updates, that they should tag you all while I'm OOO. ****
  - @steele-lm again, before you all make the new confirmation email live, can you just flag for SMEs, just to ensure that they're okay with/approve the updates.

**Comment by aliyahblackmore** [comment]
**Tracking an update here:** 

While I was OOO SMEs provided feedback. And @aprocik1 confirmed that @steele-lm  can update the confirmation email in VA notify to reflect the content in the SharePoint document - excluding the "How to submit supporting documents" section.

**Comment by aliyahblackmore** [comment]
Thanks @fiorella-io!!

And just a flag that @aprocik1 and @steele-lm got approval from SMEs to include all of the content **EXCEPT** for the content in the section "How to submit supporting documents" (we'll need to add this content later on, once we make the FDC fast-follow updates on the form and SME review again).

**Comment by fiorella-io** [comment]
Hi, @aliyahblackmore! I'm just checking to see where we are with the copy of the "How to submit supporting documents" section of the email. Thanks

**Comment by aliyahblackmore** [comment]
Hi @fiorella-io - as mentioned last week, I'm turning back to all of these updates this week (the post- 100% FDC updates and related updates to the confirmation email and static page). I'll let you all know when I share the updates with your SMEs - I'll likely have that to them by tomorrow afternoon.

**Comment by aliyahblackmore** [comment]
**Just an update here:** 

I sent a message to SMEs via Teams a few minutes ago to have them do an accuracy review of the language on the supporting documents screen in the form. I'll adjust content here once they share any feedback.

**Comment by aliyahblackmore** [comment]
**Cross posting:** 

SMEs gave the okay on a version of the medical records information. I'll track the changes in Figma next week. This isn't ready yet.

**Comment by aliyahblackmore** [comment]
**Cross posting here:** 

I'll track the content updates in the SharePoint document when I'm back on 7/8.

**Comment by fiorella-io** [comment]
Hi @aliyahblackmore, I'm checking to see if the final copy is in SharePoint so I can review it and ensure we add "reimbursement" anywhere we mention the form.

**Comment by aliyahblackmore** [comment]
Hi @fiorella-io !

- The FDC copy isn't ready yet so you all should **hold** on those FDC updates in the confirmation email. I already added a note in SharePoint just to clarify that section isn't ready yet and I flagged that on Slack earlier in the week.

- Re: adding "reimbursement,' to the email, I was just taking a look at the email subject and form name in SharePoint and I actually think that it may be okay to leave "Application for Burial Benefits (VA Form 21P-530EZ" in the confirmation email. I'd like for @aprocik1 to share their thoughts on this as well - they'll be online tomorrow morning.

**Comment by aprocik1** [comment]
@aliyahblackmore and @fiorella-io, I think it's okay to leave the confirmation email content unedited. We could consider updating the subject line to something like this: We’ve received your application for burial allowance and transportation reimbursement. But, I don't think that's necessary, since we're using the official form name as the subject line and since we're including the generated "benefits claimed" content.

**Comment by aliyahblackmore** [comment]
**Updates here on the FDC email content:** 

The most recent confirmation email content is in the [SharePoint document -](https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/Shared%20Documents/Non-disability%20Benefits/Burial/Design%20Artifacts/Burial_21P-530EZ%20Confirmation%20Email-Draft-1.0.docx?d=w54eb63d64b974104afe2f960bea55870&csf=1&web=1&e=gVKikU&nav=eyJjIjoxMDE0NTg2NDA1fQ) updates are in alignment with SME 6/5 feedback/approval on the content re: allowances for service-connected deaths and the option for the claimant to submit medical records. 

Note: These updates can happen whenever the FDC updates are going live on the online form. And also flagging that the product team should put the email through any necessary privacy reviews as we want to ensure that the email content doesn't introduce any PII.

**Comment by fiorella-io** [comment]
Thank you, @aliyahblackmore! I was able to accept your copy changes and add them to our [Google doc](https://docs.google.com/document/d/1EYAcOxEPykMe2CMo7a7zU1Alw6HcHd_8CSnkBiqce2s/edit).

**Comment by aliyahblackmore** [comment]
Thanks so much @fiorella-io !! 

- If there are any other future updates to the email, the SharePoint document will have the source of truth content - so you all should reference that. 

- And I just made some small style tweaks to what you added in Google to match what's in SharePoint exactly.
